demoapp/src/main/java/me/uport/sdk/demoapp/FingerPrintProtectionActivity.kt

```diff
@@ -0,0 +1,68 @@
+package me.uport.sdk.demoapp
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.uport.sdk.signer.UportHDSigner
+import com.uport.sdk.signer.encryption.KeyProtection
+import kotlinx.android.synthetic.main.key_protection_activity.*
+import me.uport.sdk.core.decodeBase64
+import me.uport.sdk.core.padBase64
+import me.uport.sdk.core.toBase64
+import org.walleth.khex.toHexString
+import java.util.*
+
+
+class FingerPrintProtectionActivity : AppCompatActivity() {
+
+    private var address: String = ""
```

Comment:
Even though this is an eth address it is not used as such, it's only used to identify the seed.
It should be renamed to `seedHandle`.
In the near future this handle may become more than an address or may even be created by the user.